src/app/components/CheckoutDialog.tsx

```diff
@@ -152,20 +152,21 @@ function CheckoutDialog(props: any) {
                     id="nameInput"
                     label="Name"
                     variant="standard"
-                    defaultValue={`${user.firstName} ${user.name}`}
```

Comment:
Hab gerade nochmal geschaut, technisch wird in einer Buchung nur der User hinterlegt, also auch keine anderen Adressen. Wir können also entweder beim Checkout nur noch 2 Optionen: 
- Login, wenn nicht eingeloggt
- Abschicken, wenn eingeloggt (und die adressdaten sind egal, weil bei der buchung ja der User mitgegeben wird)